tests/gtest/avifincrtest_helpers.cc

```diff
@@ -364,10 +362,7 @@ avifResult DecodeIncrementally(const avifRWData& encoded_avif,
              previously_decoded_row_count, decoded_row_count);
       AVIF_CHECKERR(false, AVIF_RESULT_INVALID_ARGUMENT);
     }
-    bool has_gain_map = false;
-#if defined(AVIF_ENABLE_GAIN_MAP)
-    has_gain_map = (reference.gainMap != nullptr);
-#endif
+    const bool has_gain_map = (reference.gainMap != nullptr);
```

Comment:
Nit: Inline this variable. See the use of `reference.alphaPlane != nullptr` in line 367.